### ISSUE TITLE
Modify train-station preset, potential add railway-station buiding preset

### ISSUE THREAD

**Issue by sun-geo** [openstreetmap/id-tagging-schema#59]
It seems there is a current preset train-station active, which adds the following tags 
for an area object:
`building=yes`
`public_transport=station`
`railway=station`
`train=yes`

for a node object:
`public_transport=station`
`railway=station`
`train=yes`

Even for area objects it seems there should be no `building` added compare to wiki example picture:
https://wiki.openstreetmap.org/wiki/Tag:railway=station#Railway_Station_Layout

Mayba a separate preset for a railway-station (building)
https://wiki.openstreetmap.org/wiki/Tag:building%3Dtrain_station
could compensate a gap when the `building` tag would be removed from the train-station preset?

**Comment by peternewman** [comment]
The preset already exists:
https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/railway/_station.json

It's just it's not searchable for some reason:
https://github.com/openstreetmap/id-tagging-schema/blob/bf95421f65018ccd7f398f8459b37462febddcba/data/presets/railway/_station.json#L18-L19

Possibly to stop people tagging actual stations as just the building?

**Comment by tyrasd** [comment]
> It's just it's not searchable […] Possibly to stop people tagging actual stations as just the building?

Yes, that seems like it was the intention, see https://github.com/openstreetmap/iD/issues/5903#issuecomment-464810972 or https://github.com/openstreetmap/iD/pull/2116/files#r9478481

I think this is similar to the school presets: there is the "actual" (amenity=) school preset and there is the school building preset. Both are searchable. Since the naming is more precise now, I think we could make the train station building preset searchable.
